CommonMark/render/rst.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals
+
+
+from CommonMark.render.renderer import Renderer
+
+
+class ReStructuredTextRenderer(Renderer):
+    def __init__(self, indent_char=' '):
+        self.indent_char = indent_char
+        self.indent_length = 0
+
+    def lit(self, s):
+        if s == '\n':
+            indent = u''  # Avoid whitespace if we're just adding a newline
```

Comment:
You can just use `''` instead of `u''`, since we have `from __future__ import unicode_literals` at the top here - all string literals are unicode.